include/packager/live_packager.h

```diff
@@ -70,10 +71,22 @@ struct LiveConfig {
     VIDEO,
   };
 
+  enum class EncryptionScheme {
+    NONE,
+    SAMPLE_AES,
+    AES_128,
+  };
+
   OutputFormat format;
   TrackType track_type;
   // TOOD: do we need non-integer durations?
   double segment_duration_sec;
+
+  // TODO: should we allow for keys to be hex string?
+  std::vector<uint8_t> iv;
+  std::vector<uint8_t> key;
+  std::vector<uint8_t> key_id;
```

Comment:
Someone ought to convert them. We receive strings from KMS, as far as I know

In terms of API, SWIG maps vectors to slices, so it is probably easier to do conversion on Go side and just pass slices I guess?

Reply:
I believe we should keep this interface and have Go pass them in as a vector of bytes.  

@jedekar I am afraid Go slices don't map to vectors directly, so there'll still be a small conversion.